dio/lib/src/dio_mixin.dart

```diff
@@ -706,13 +701,13 @@ abstract class DioMixin implements Dio {
   }
 
   static Response<T> assureResponse<T>(
-    Object response, [
-    RequestOptions? requestOptions,
-  ]) {
+    Object response,
+    RequestOptions requestOptions,
+  ) {
```

Comment:
I cannot evaluate the affection by either breaking this or making it `@internal`. However, an exposed API should be considered as used by everyone. Can we keep the method and add another underscored method to pass through?